cucoslib/workers/recommender.py

```diff
@@ -257,6 +258,31 @@ def get_reference_stacks_from_graph(self, list_packages):
 
         return list_ref_stacks
 
+    def get_version_information(self, input_list, ecosystem):
+        """Fetch the version information for each of the packages"""
+        input_packages = []
+        for package in input_list:
+            input_packages.append(package)
```

Comment:
Then why there's `list` in its name ?

You can use list comprehension in any case:

```python
input_packages = [package for package in input_list]
```